examples/experimental/terrain/src/app.js

```diff
@@ -20,16 +17,16 @@ const INITIAL_VIEW_STATE = {
   bearing: 140,
   pitch: 60
 };
-
 // Constants
 const TERRAIN_RGB = 'https://api.mapbox.com/v4/mapbox.terrain-rgb';
 
-const getTerrainData = ({x, y, z}) => {
-  const terrainTile = `${TERRAIN_RGB}/${z}/${x}/${y}.pngraw?access_token=${MAPBOX_TOKEN}`;
-  // Some tiles over the ocean may not exist
-  // eslint-disable-next-line handle-callback-err
-  return load(terrainTile).catch(err => null);
-};
+function getTerrain(tile) {
```

Comment:
Let's move this to a test. As a public-facing example, I prefer to keep things simple.

Reply:
Are you referring to the two ways this function can be used, and reducing it to the simple TileLayer usage?

That’s fair since this becomes the website example which will just show one way.

I was going to prepare an example of how I’d typically use it without TileLayer and put it somewhere too, maybe explore some fun advance usages. Any suggestions?

Reply:
You can put the simple usage example in  https://github.com/uber/deck.gl/blob/master/website/src/components/demos/layer-demos.js — these are the examples embedded in the layer docs.

Reply:
Ok cool! I’ll merge this and get into geo-layers PR which can have tests and layer-demos example.